### ISSUE TITLE
job link in url is pointing to the home page

### ISSUE THREAD

**Issue by Deep-Codes** [fireship-io/fireship.io#489]
I think this is pretty understandable.

let me know if more context is needed

<img width="539" alt="Screenshot 2021-09-29 at 9 44 39 PM" src="https://user-images.githubusercontent.com/61158210/135308199-a410d2a3-9ab4-4154-9cf8-861a45afc338.png">

**Comment by szilvesztercsab** [comment]
Yeah [it does](https://github.com/fireship-io/fireship.io/blob/c49643120e835e90a20a0550dd2e751853fc30ab/hugo/layouts/partials/footer.html#L14).
